src/oidcop/__init__.py

```diff
@@ -1,6 +1,6 @@
 import secrets
 
-__version__ = "2.3.4"
+__version__ = "2.3.5"
```

Comment:
Considering that we have an additional feature like iss identification, would we move to 2.4.0 instead or 2.3.5 if good right now?

Reply:
OK